src/sentences/utils.rs

```diff
@@ -200,6 +200,79 @@ mod tests {
         assert_eq!(time.nanosecond(), 500_000_000);
     }
 
+    #[test]
+    #[should_panic]
+    fn test_parse_hms_fail_1() {
+        use chrono::Timelike;
+        let (_, time) = parse_hms("247090,").unwrap();
+        assert_eq!(time.hour(), 24);
+        assert_eq!(time.minute(), 70);
+        assert_eq!(time.second(), 90);
+        assert_eq!(time.nanosecond(), 0);
+        let (_, time) = parse_hms("247090.8,").unwrap();
+        assert_eq!(time.hour(), 24);
+        assert_eq!(time.minute(), 70);
+        assert_eq!(time.second(), 90);
+        assert_eq!(time.nanosecond(), 800_000_000);
+    }
+
+    #[test]
+    #[should_panic]
+    fn test_parse_hms_hr_fail() {
+        use chrono::Timelike;
+        let (_, time) = parse_hms("244550,").unwrap();
+        assert_eq!(time.hour(), 24);
+        assert_eq!(time.minute(), 45);
+        assert_eq!(time.second(), 50);
+        assert_eq!(time.nanosecond(), 0);
+    }
+
+    #[test]
+    #[should_panic]
+    fn test_parse_hms_min_fail() {
+        use chrono::Timelike;
+        let (_, time) = parse_hms("237050,").unwrap();
+        assert_eq!(time.hour(), 23);
+        assert_eq!(time.minute(), 70);
+        assert_eq!(time.second(), 50);
+        assert_eq!(time.nanosecond(), 0);
```

Comment:
If calling `parse_hms(...).unwrap()` will panic the rest of the assertions will not be executed.
It would be best to assert the expected error instead of `unwrap`ing in this case.
This way you can be sure that if `minutes` is out of bound, then you get an `minutes` error not something else.

Right now, you cannot be sure if the minutes are the culprit of the error.